dsalgo/string_algo.py

```diff
@@ -0,0 +1,156 @@
+class Strings:
+    def encode(self, strs):
+        """
+        Encodes a list of strings to a single string.
+        params: List[str]
+        returns: str
+        """
+        res = ""
+        for string in strs.split():
+            res += str(len(string)) + ":" + string
+        return res
+
+    def decode(self, s):
+        """
+        Decodes a single string to a list of strings.
+        params: str
+        returns: List[str]
+        """
+        strs = []
+        i = 0
+        while i < len(s):
+            index = s.find(":", i)
+            size = int(s[i:index])
+            strs.append(s[index+1:index+1+size])
+            i = index + 1 + size
+        return strs
+
+    def fizzbuzz(self, n):
+        """
+        params : N int
+        returns: string
+
+        A function that returns an array containing
+        the numbers from 1 to N,
+        Where N (size of an array) is the parametered value.
+        N will never be less than 1.
+        There is no fancy algorithm to solve fizz buzz.
+        Iterate from 1 through n
+        Use the mod operator to determine if the current
+        iteration is divisible by:
+        3 and 5 -> 'FizzBuzz'
+        3 -> 'Fizz'
+        5 -> 'Buzz'
+        else -> string of current iteration
+        return the results
+        Complexity:
+        Time: O(n)
+        Space: O(n)
+        """
+        # Validate the input
+        if n < 1:
+            raise ValueError("n cannot be less than one")
+        if n is None:
+            raise TypeError("n cannot be None")
```

Comment:
Remove this function. @codesankalp @ravi5175 I don't think that it is valuable to us in any sense. What are your views?